.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
-repos:
-  - repo: https://github.com/psf/black
-    rev: 19.3b0
-    hooks:
-      - id: black
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black

```

Comment:
not sure why this one has changed?

Reply:
no idea either
doesn't look like anything's changed